src/include/miopen/cat/problem_description.hpp

```diff
@@ -120,13 +143,21 @@ struct ProblemDescription : ProblemDescriptionBase
         {
             if(!xDescs[i]->IsPacked())
             {
+#if MIOPEN_BUILD_DEV || !MIOPEN_NDEBUG
+                MIOPEN_THROW(miopenStatusBadParm, "CatForward: Unpacked tensors not supported.");
+#else
                 return false;
+#endif
             }
         }
 
         if(!yDesc.IsPacked())
         {
+#if MIOPEN_BUILD_DEV || !MIOPEN_NDEBUG
+            MIOPEN_THROW(miopenStatusBadParm, "CatForward: Unpacked tensors not supported.");
```

Comment:
Ditto

Reply:
remove exception, It returns false if it is not packed.